pennylane/templates/subroutines/qsvt.py

```diff
@@ -21,131 +21,203 @@
 
 import pennylane as qml
 from pennylane.operation import AnyWires, Operation
-from pennylane.ops import BlockEncode, PCPhase
 from pennylane.ops.op_math import adjoint
 from pennylane.queuing import QueuingManager
 from pennylane.wires import Wires
 
 
-def qsvt(A, angles, wires, convention=None):
-    r"""Implements the
-    `quantum singular value transformation <https://arxiv.org/abs/1806.01838>`__ (QSVT) circuit.
+def qsvt(A, poly, encoding_wires, block_encoding=None):
```

Comment:
It could be an option :) 
In that case it would be enough to update `qml.PrepSelPrep` to make `qml.Qubitization` and `qsvt` work. Let's leave it out of this PR